api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -20,71 +27,89 @@
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@Import({InstitutionServiceImpl.class})
+@Import({InstitutionServiceImpl.class, InstitutionMapperImpl.class})
 public class InstitutionServiceTest {
   @Autowired private InstitutionService service;
+  @Autowired private UserDao userDao;
+  @Autowired private VerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation;
 
-  private final Institution testInst =
+  private final Institution TEST_INST =
       new Institution().shortName("test").displayName("this is a test");
 
-  // the DB converts nulls to empty lists
-  private final Institution testInstAfterRT =
+  // the mapper converts nulls to empty sets
+  private final Institution TEST_INST_AFTER_RT =
       new Institution()
-          .shortName(testInst.getShortName())
-          .displayName(testInst.getDisplayName())
+          .shortName(TEST_INST.getShortName())
+          .displayName(TEST_INST.getDisplayName())
           .emailDomains(Collections.emptyList())
           .emailAddresses(Collections.emptyList());
 
   @Before
   public void setUp() {
-    service.createInstitution(testInst);
+    // will be retrieved as TEST_INST_AFTER_RT
+    service.createInstitution(TEST_INST);
   }
 
   @Test
   public void test_createInstitution() {
-    assertThat(service.getInstitutions()).hasSize(1);
+    assertThat(service.getInstitutions()).containsExactly(TEST_INST_AFTER_RT);
 
-    service.createInstitution(
-        new Institution().shortName("otherInst").displayName("The Institution of testing"));
-    assertThat(service.getInstitutions()).hasSize(2);
+    final Institution otherInst =
```

Comment:
It's not obvious to me what this addition to the test case is checking. It feels like we've actually munged two tests into one here, which can sometimes be a net negative for clarity / maintainability. Either add a comment to clarify why this is part of the "base" createInstitution test method, or otherwise split out to a separate method with a name which clarifies the intent.

Reply:
renamed to "createAnother" - is that clear?